etc/LATISS/primary_hdu.header

```diff
@@ -40,11 +40,11 @@ RAEND   =                      / RA of telescope from AZEND and ELEND
 DECEND  =                      / DEC of telescope from AZEND and ELEND
 ROTPA   =                      / Rotation angle relative to the sky (deg)
 ROTCOORD= 'sky'                / Telescope Rotation Coordinates
-HASTART =                      / [HH:MM:SS] Telescope hour angle at start
+HASTART =                      / [deg] Telescope hour angle at start
 ELSTART =                      / [deg] Telescope zenith distance at start
 AZSTART =                      / [deg] Telescope azimuth angle at start
 AMSTART =                      / Airmass at start
-HAEND   =                      / [HH:MM:SS] Telescope hour angle at end
+HAEND   =                      / [deg] Telescope hour angle at end
```

Comment:
This is decimal hours not degrees.

Reply:
Thanks for noticing, changed to `[decimal hrs]`